test/cmdlineTests/standard_missing_type_useLiteralContent/input.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Shouldn't the file be named "standard_missing_key_..."?

Reply:
Yep, changed the name.